README.md

```diff
@@ -1,5 +1,19 @@
 # inHEARTive
 
+### Detox for React Native
+
+#### pre requirements
+
+`brew tap wix/brew`
+
+`brew install applesimutils`
```

Comment:
Please use

```
```shell
$ brew tap wix/brew
$ brew tap wix/brew
$ npm i -g jest # install jest globally
\```
```
to show shell commands (I mean to use triple backtick block and command prompt which shows common user should use those commands).